tools/performance/benchmark-analysis/README.md

```diff
@@ -0,0 +1,5 @@
+# Analysing benchmarks
+
+This workflow is prepared mostly to analyse the output of benchmarks from
+`test/Exploratory_Benchmarks`. See `test/Exploratory_Benchmarks/README.md` for
+more information.
```

Comment:
I believe this works for any benchmark run?

Reply:
Like I [commented before](https://github.com/enso-org/enso/pull/7270#discussion_r1266421686)